docs/_includes/ds6/pagination.html

```diff
@@ -0,0 +1,176 @@
+<div id="pagination">
+    <h2>Pagination</h2>
+    <p>Pagination allows a user to navigate back and forwards through a URL based dataset, or jump directly to any specific
+        URL in that set.</p>
+    <p>Pagination links may update the results immediately on the client via AJAX, or on the server via a full page reload.
+        <p>
+
+            <svg hidden>
+                <symbol id="svg-icon-arrow-left" viewBox="0 0 16 16">
+                    <path d="M3.1221 7.2116s4.2752-4.0858 4.601-4.3978c.3257-.312.2865-.8252 0-1.1-.2875-.2747-.8459-.2953-1.15 0-.3052.2954-6.5732 6.2828-6.5732 6.2828s6.1954 5.903 6.5732 6.2827c.3777.3797.8566.2816 1.15 0 .2943-.2806.3718-.728 0-1.099-.372-.3708-4.601-4.3978-4.601-4.3978h10.0241c.578 0 .8537-.3924.8537-.786 0-.3924-.2728-.7849-.8537-.7849H3.1222z"></path>
+                </symbol>
+                <symbol id="svg-icon-arrow-right" viewBox="0 0 16 16">
+                    <path d="M10.8778 8.8173s-4.2752 4.0858-4.6 4.3978c-.3267.313-.2875.8242 0 1.1.2865.2747.8449.2953 1.149 0C7.732 14.0197 14 8.0323 14 8.0323s-6.1944-5.904-6.5732-6.2837c-.3777-.3787-.8556-.2806-1.149 0-.2943.2816-.3728.729 0 1.1.371.3709 4.6 4.3978 4.6 4.3978H.8546c-.5789 0-.8546.3925-.8546.786 0 .3924.2728.785.8546.785h10.0232z"></path>
+                </symbol>
+            </svg>
+            <div class="demo">
+                <div class="demo__inner">
+                    <nav class="pagination" aria-labelledby="pagination-heading" role="navigation">
+                        <span aria-live="polite" role="status">
+                            <h2 class="clipped" id="pagination-heading">Results Pagination - Page 1</h2>
+                        </span>
+                        <a aria-disabled="true" aria-label="Previous Page" class="pagination__previous" href="http://www.ebay.com/sch/i.html?_nkw=guitars">
+                            <svg aria-hidden="true" focusable="false" width="16" height="16">
```

Comment:
Let's use background SVG on an empty span as the default documented case for the icons. We can keep support for inline SVG too (using `.pagination__previous span` and `.pagination__previous svg`)

Reply:
Okay I will change them to `span` although the design for `ebay-icon` would have to change a little to incorporate the background SVG